src/wp-includes/option.php

```diff
@@ -799,6 +803,11 @@ function update_option( $option, $value, $autoload = null ) {
 		return false;
 	}
 
+	// if the value changed, we still need to validate that this isn't the value that's in DB already now
```

Comment:
```suggestion
	// if the value changed, we still need to validate that this isn't the value that's in DB already now.
```

Reply:
@kkmuffme The WordPress-Docs ruleset is not currently enabled in core's PHPCS config, so this kind of issue is not reported on CI. Hence the comment here :)

Reply:
The main reason is that the WordPress-Docs ruleset is far from complete/perfect, so there would be too many false positives or things being missed.

See https://github.com/WordPress/gutenberg/discussions/59786 for a current discussion.